src/guessRegister.js

```diff
@@ -0,0 +1,21 @@
+export class GuessRegister {
```

Comment:
I would `export default class GuessRegister` because it's the only thing you're exporting from here this file. So it can be default.